GOVERNANCE.md

```diff
@@ -0,0 +1,57 @@
+# k8gb Governance
+
+This document defines governance policies for k8gb project.
+
+## Principles
+
+The k8gb project community adheres to the following principles:
+
+- Open: k8gb is open source.
+- Welcoming and inclusive: See our [Code of Conduct](#code-of-conduct)
+- Transparent and accessible: Any changes to k8gs source code and collaborations on the project are publicly accessible (GitHub issues, PRs, and discussions).
+- Merit: Ideas and contributions are accepted according to their technical merit and alignment with project objectives, scope, and design principles.
+
+
+## Code of Conduct
+
+k8gb follows [Code of Conduct](./code-of-conduct.md), which is aligned with the [CNCF Code of Conduct](https://github.com/cncf/foundation/blob/master/code-of-conduct.md).
+
+## Contributors
+
+Anyone can contribute to the project. More about contribution rules and technical aspects can be found in the [CONTRIBUTING](CONTRIBUTING.md) documentation.
+## Maintainers
+
+Maintainers are defined in the [CODEOWNERS](CODEOWNERS) file by their GitHub user handles.
+New maintainers are nominated by existing maintainers and are elected by a supermajority of votes of existing maintainers.
+If the maintainer is no longer be able to perform the maintainer duties, they should volunteer to be moved to emeritus status.
+Maintainers can be also removed from active maintainers list by a supermajority of votes of existing maintainers.
+
+Below is the list of maintainer's general responsibilities in additions to those defined for project contributors:
+
+- Triages GitHub issues
+- Reviews pull requests
+- Runs k8gb releases
+- Forms k8gb architecture, strategy and roadmap
+- Actively supports k8gb community in related channels
+- Represents k8bg project in public media (events, slack channels, etc)
+- Is involved in CNCF-related activities and projects
+- Ensures CNCF guidelines compliancy
+- Owns project infrastructure
+
+## Project Lead
+
+Project Lead is a project maintainer with final vote on project decisions
+
+## Conflict Resolutions
+
+In most cases, it is expected that conflicts are amicably resolved by involved parties.
+In case of escalation, conflicts are resolved by voting, with ruling of supermajority of votes of project maintainers. Project lead has final vote on ruling.
```

Comment:
```suggestion
In case of escalation, conflicts are resolved by voting, with the ruling of a supermajority of votes of project maintainers. The project lead has the final vote on the ruling.
```